common/djangoapps/third_party_auth/models.py

```diff
@@ -500,6 +500,7 @@ class LTIProviderConfig(ProviderConfig):
     )
 
     lti_hostname = models.CharField(
+        default='localhost',
```

Comment:
I notice that the last migration had `default='localhost'` for `lti_hostname` but the model did not.

@dianakhuang Can you comment on whether we want this default or not?